ExtractData/PKHeX.ExtractData/ExtTMHM3.cs

```diff
@@ -0,0 +1,376 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PKHeX.ExtractData
+{
+    class ExtTMHM3 : ExtMoveTutors4
+    {
+        internal override GenerationData GData => Generation3Data.Create();
+
+        static readonly string[] TMMoves_Gen4 =
```

Comment:
I think it would be better to store this list of strings as an embedded resource.  Not only is it easier to manage, but it makes translation a _whole_ lot easier.

Also this variable has no references.